oracle/src/lib.rs

```diff
@@ -72,18 +72,13 @@ impl<T: Trait> Module<T> {
 			.collect()
 	}
 
-	pub fn get(key: &T::Key) -> Option<TimestampedValue<T::Value, MomentOf<T>>> {
+	pub fn get(key: &T::Key) -> Option<T::Value> {
 		if <HasUpdate<T>>::exists(key) {
```

Comment:
`<HasUpdate<T>>::take(&key)` will read and remove the value so no need to insert false later